code/ai/aicode.cpp

```diff
@@ -13592,6 +13591,14 @@ void ai_execute_behavior(ai_info *aip)
 			aip->mode = AIM_NONE;
 		}
 		break;
+	case AIM_GET_BEHIND:
+		if (En_objp) {
```

Comment:
I recommend an `nprintf` indicating that this mode was triggered, so that if there is a future AI bug which requires a really deep investigation, we can spot it that this mode is being triggered, which might be contributing to it.

Reply:
Hmm.  Ok.